res/controllers/Traktor-Kontrol-S4-MK2-hid-scripts.js

```diff
@@ -113,6 +138,7 @@ TraktorS4MK2.registerInputPackets = function() {
   MessageShort.addControl("[EffectRack1_EffectUnit1_Effect1]", "!FXButton", 0x12, "B", 0x40);
   MessageShort.addControl("[EffectRack1_EffectUnit1_Effect2]", "!FXButton", 0x12, "B", 0x20);
   MessageShort.addControl("[EffectRack1_EffectUnit1_Effect3]", "!FXButton", 0x12, "B", 0x10);
+  MessageShort.addControl("[EffectRack1_EffectUnit2]", "!FXFocusButton", 0x11, "B", 0x02);
```

Comment:
This is just a guess.

Reply:
This should be the same as line 137 `0x12, "B", 0x80`

Reply:
Also should be `[EffectRack1_EffectUnit1]`